authorizer-app-backend/src/main/java/org/radarbase/authorizer/api/Project.kt

```diff
@@ -28,7 +28,7 @@ fun MPProject.toProject() = Project(
         name = name,
         location = location,
         organization = organization,
-        description = description,
+        description = description
```

Comment:
The comma was intentional. You can set this as default behaviour in IntelliJ IDEA by going to Editor -> Code Style -> Kotlin -> Other tab -> enable trailing comma.